### ISSUE TITLE
Let users add a windows chess engine

### ISSUE THREAD

**Issue by Nazine** [pychess/pychess#943]
When adding a .sh script that redirects to wine for it to open a windows chess engine, pychess hangs horribly and sometimes even crashes.

A sample script;
# !/bin/sh

wine "Houdini15.exe"

When adding Houdini 1.5a, PyChess freezes.

**Comment by gbtami** [comment]
This will not work because PyChess uses self.process = suprocess.Popen(...) to start the selected engine and then self.process.communicate(...) to to send "uci" or "xboard" to detect the protocol it supports. But after starting the shell script it will do nothing with "uci", it will not pass it to houdini. 

If you want to add a windows engine to PyChess, add this to your ~/.config/pychess/engines.json file:

```
 {
  "command": "/full_path_to/Houdini15.exe", 
  "country": "be", 
  "name": "Houdini.exe", 
  "protocol": "uci", 
  "vm_command": "/usr/bin/wine", 
  "vm_name": "wine"
 }, 
```

**Comment by gbtami** [comment]
I think fix for #959 fixed this too. No need to create .sh scripts for .exe engines. PyChess will add wine as vm for them.
